linera-views/src/reentrant_collection_view.rs

```diff
@@ -685,7 +740,8 @@ where
                 hasher.update_with_bcs_bytes(&keys.len())?;
                 for key in keys {
                     hasher.update_with_bytes(&key)?;
-                    let view = self.try_load_entry(key).await?;
+                    // We can unwrap since the view is loaded from the keys, so we know it is present
+                    let view = self.try_load_entry(key).await?.unwrap();
```

Comment:
Rewritten as "We can unwrap since key is in keys, so we know it is present."

Reply:
nit: `key` and `keys` (with quotes). Note that this works because the list of keys cannot be mutated any longer by other parts of the code using `&self`.

Reply:
Ok.